.github/workflows/build-release-binaries.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Build release binaries"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_binaries:
+    strategy:
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            archive_ext: tar
+          - target: x86_64-apple-darwin
+            os: macos-latest
+            archive_ext: tar
+          - target: x86_64-pc-windows-msvc
+            os: windows-latest
+            archive_ext: zip
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout tagged commit
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.release.target_commitish }}
+
+      - uses: Swatinem/rust-cache@v1.3.0
+
+      - name: Build ${{ matrix.target }} release binary
+        run: cargo build --target=${{ matrix.target }} --release
+
+      # Remove once python 3 is the default
+      - uses: actions/setup-python@v2.2.2
+        with:
+          python-version: '3.x'
+
+      - id: create-archive-name
+        shell: python # Use python to have a prettier name for the archive on Windows.
+        run: |
+          import platform
+          os_info = platform.uname()
+
+          arch = os_info.machine
+
+          triple = "${{ matrix.target }}".split("-")
+          arch = triple[0]
```

Comment:
Does `arch` here always overwrite the `arch` on line 44?

Reply:
Uh yeah, I think you are right!
Apologies, I copied this from one of our projects :)

Will fix!